src/main/java/org/swisspush/redisques/QueueStatsService.java

```diff
@@ -225,7 +229,7 @@ public static class Queue {
         private Long lastDequeueAttemptEpochMs;
         private Long lastDequeueSuccessEpochMs;
         private Long nextDequeueDueTimestampEpochMs;
-        private Queue(String name, long size){
+        Queue(String name, long size){
```

Comment:
Do not change visibility of methods/variables just to test them. This is bad practice. If you can find a workaround or there is a arguable reason, it may be ok.